src/sparsevector.jl

```diff
@@ -47,3 +47,40 @@ function findnz(vec::SparseVector{T}) where T
     V = to_jl_type(T)[vec[idx] for idx in I]
     return (I, V)
 end
+
+# implementation of SparseVector{Bool} with Int64 length using a polymake Set
+struct SparseVectorBool <: SparseVector{Bool}
+    l::Int64
+    s::Set{to_cxx_type(Int64)}
+end
+
+spzeros(::Type{Bool}, n::Base.Integer) = SparseVectorBool(n, Polymake.Set{to_cxx_type(Int64)}())
+
+Base.size(v::SparseVector{Bool}) = (v.l,)
+Base.eltype(::SparseVector{Bool}) = Bool
+
+Base.show(io::IOContext, ::MIME{Symbol("text/plain")}, v::Polymake.SparseVectorBool) = print(io, length(v), "-element SparseVectorBool. `true` for indices:\n", v.s...)
+
+Base.@propagate_inbounds function Base.getindex(V::SparseVector{Bool}, n::Base.Integer)
+    (V.l >= n && n >= 1) || throw(BoundsError(V, n))
```

Comment:
```suggestion
    @boundscheck checkbounds(V, n)
```
(that is assuming that `SparseVector<:AbstractArray`)